src/compiler/commandLineParser.ts

```diff
@@ -7,6 +7,39 @@
 namespace ts {
     /* @internal */
     export const compileOnSaveCommandLineOption: CommandLineOption = { name: "compileOnSave", type: "boolean" };
+
+    /* @internal */
+    export const libMap = createMapFromTemplate({

```

Comment:
This just maps `x` to `lib.x.d.ts`. Could be a function instead, combined with a *set* of libs.